### ISSUE TITLE
Compatibility with latest YouTube API changes re vevo-like videos

### ISSUE THREAD

**Issue by DabKick** [HaarigerHarald/android-youtubeExtractor#1]
Really appreciate this code, have used it in our app and it is truly awesome. Need a fix - URL for Vevo-like videos are no longer extractable. It would be great to have this fixed so we can extract URLs for vevo-like videos with signatures which is crucial for our work. If you can point to code snippets to accomplish that, that'd be great too.
thanks again for this great code. Fantastic work.

**Comment by HaarigerHarald** [comment]
Signature deciphering should be up to date now with e61ed9d. Are you on the latest release?

Should the problem persist please enable the LOGGING flag in the YouTubeUriExtractor class, run the test cases and send me the logs in case they fail.

**Comment by DabKick** [comment]
Yes on the latest release...thanks, it all works! iOS had to go through an update v recently re vevo videos - should we expect another android update too? Thanks so much for your response.
